src/__tests__/unit/cache-leveldb-real-data.test.ts

```diff
@@ -0,0 +1,14 @@
+import { LevelDbCache } from '../../cache/impl/LevelDbCache';
+import { defaultCacheOptions, WarpFactory } from '../../core/WarpFactory';
+
+jest.setTimeout(90000)
+describe('Prune real Lmdb cache. Copy the real cache to ./dst, unskip and run yarn test:cache:real', () => {
```

Comment:
I'm not sure that this test should be run by default...the tests are already executing for quite long and the timeout above looks kinda scary ;-)

Reply:
I guess `.skip` is missing here.

Reply:
Ah, yes. Fixed.